app/uk/gov/hmrc/incorporatedentityidentification/testonly/GetCtReferenceStubController.scala

```diff
@@ -91,6 +91,22 @@ class GetCtReferenceStubController @Inject()(controllerComponents: ControllerCom
     "234568OO" -> "2177007924",
```

Comment:
I know this is not part of your change but this key "234568OO" has double letter O and I believe it should be double zero, but maybe not